src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+        <precision>2</precision>
+        <required>true</required>
+        <scale>0</scale>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Label__c</fullName>
+        <description>The custom label api name used in the Primary Button on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Primary Button Label</inlineHelpText>
+        <label>Primary Button Label</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Type__c</fullName>
+        <description>The type of action related the Primary Button</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The type of action related the Primary Button</inlineHelpText>
+        <label>Primary Button Type</label>
+        <required>true</required>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Disabled</fullName>
+                    <default>true</default>
+                    <label>Disabled</label>
+                </value>
+                <value>
+                    <fullName>sfdc:new-tab</fullName>
+                    <default>false</default>
+                    <label>SFDC New Tab</label>
+                </value>
+                <value>
+                    <fullName>link</fullName>
+                    <default>false</default>
+                    <label>External Page</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Value__c</fullName>
+        <description>The values is used to for the action to the Primary Button, In case is an External Page Button is the external URL and the case is SFDC New Tab is the api name to the tab to open format {type(without started &amp;standard__&amp;)}:{objectApiName}:{actionName}</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The value used for the action of the Primary Button</inlineHelpText>
+        <label>Primary Button Value</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Secondary_Button_Label__c</fullName>
+        <description>The custom label api name used in the Secondary Button on a section item on the Getting Started page</description>
```

Comment:
I recommend changing this to "The custom label api name for the Secondary Button in a section item on the Getting Started page."